src/OrchardCore.Modules/OrchardCore.OpenId/Recipes/OpenIdServerSettingsStepModel.cs

```diff
@@ -0,0 +1,40 @@
+using System.ComponentModel.DataAnnotations;
+using System.Security.Cryptography.X509Certificates;
+using static OrchardCore.OpenId.Settings.OpenIdServerSettings;
```

Comment:
Why we need this?

Reply:
The TokenFormat enum is defined in OpenIdServerSettings class, so the using static to the class is needed.

Reply:
Ok, coz I didn't see `OpenIdServerSettings` usage in the code